Week5/app/src/main/java/com/isabelledionisius/mymovieapp/MovieRepository.kt

```diff
@@ -0,0 +1,23 @@
+package com.isabelledionisius.mymovieapp
+
+import androidx.lifecycle.LiveData
+
+class MovieRepository(private val movieDao: DaoMovie) {
+
+    var wordRepository = WordRepository(movieDao)
+
+    // Declares the DAO as a private property in the constructor. Pass in the DAO
+// instead of the whole database, because you only need access to the DAO
+    class WordRepository(private val movieDao: DaoMovie) {
```

Comment:
Why do you need the `WordRepository`? What does it even represent, as you just wrapped it with the MovieRepository, which has the same thing.

I think this is copied & pasted from somewhere else, and this is something we are strongly against.